pom.xml

```diff
@@ -78,7 +83,12 @@
 			<dependency>
 				<groupId>org.apache.hadoop</groupId>
 				<artifactId>hadoop-client</artifactId>
-				<version>1.0.3</version>
+				<version>2.6.0</version>
```

Comment:
Fixed, except that it seems I have to put hadoop dependency in camus-api/pom.xml, otherwise mvn clean package fails.